test/initial_values.jl

```
@@ -172,7 +172,7 @@ end
             [p => 2q, q => 3p]; warn_cyclic_dependency = true)
     catch
     end
-    @test_throws ModelingToolkit.UnexpectedSymbolicValueInVarmap ODEProblem(
+    @test_throws ModelingToolkit.MissingGuessError ODEProblem(
```

This is one of those cases that I'm not sure about, should this throw the MissingGuessError?

---

Yes, p and q would be missing and need a guess.